ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource

```diff
@@ -40,14 +40,18 @@ Fill In Run Creation Form    # robocop: disable
     [Documentation]    Compiles the form to create a pipeline run.
     ...                It works when you start server creation from either
     ...                DS Project details page or DS Pipelines page.
-    [Arguments]    ${name}    ${pipeline_name}    ${from_actions}=${TRUE}    ${run_type}=Immediate
-    ...            ${trigger_type}=Periodic    ${start_date}=${NONE}    ${start_time}=${NONE}
+    [Arguments]    ${name}    ${pipeline_name}    ${from_actions_menu}    ${from_actions}=${TRUE}
+    ...            ${run_type}=Immediate    ${trigger_type}=Periodic    ${start_date}=${NONE}    ${start_time}=${NONE}
     ...            ${end_date}=${NONE}    ${end_time}=${NONE}    ${cron_expr}=${NONE}&{model_param}
     Input Text    ${PIPELINE_NAME_INPUT_XP}    ${name}
     Input Text    ${PIPELINE_DESC_INPUT_XP}    ${name}
-    Run Keyword And Continue On Failure
-    ...    Element Should Be Disabled    ${GENERIC_CREATE_BTN_XP}
-    Select Pipeline For The Run    pipeline_name=${pipeline_name}
+    IF    ${from_actions_menu} == ${FALSE}
+        Wait Until Page Contains Element    xpath://button[span[text()='${pipeline_name}']]
+    ELSE
+        Run Keyword And Continue On Failure
+        ...    Element Should Be Disabled    ${GENERIC_CREATE_BTN_XP}
+        Select Pipeline For The Run    pipeline_name=${pipeline_name}
```

Comment:
hm probably, in order to warn when the issue happens, we could use a different flow like:

```
IF ${GENERIC_CREATE_BTN_XP} is enabled when it should not
THEN raise a WARN message and Wait until the dropdown gets the value
```